src/overview/woodcutting/LogDisplay.tsx

```diff
@@ -9,6 +10,9 @@ const id = "LogDisplay";
 const LogDisplay = ({ log }: Props) => {
   const [amount] = useNumberItemObserver(log, id);
 
+  const onLogClick = (event: MouseEvent) => {
```

Comment:
I agree, a tootip is important here.
Using the modal here was a conscious decision tho. I think it's super important to have destructive decisions like "burning all available logs" guarded by a confirmation dialog. 
As suggested I added alternative modes to the shortcuts tho. I think this way we have the best of both worlds.

As a side note. Personally I would prefer to have it like this with the bags as well, since I already miss clicked once and opened bags I didn't want to open. :) Maybe we can change that an other time.